cmd/gogio/androidbuild.go

```diff
@@ -867,6 +874,17 @@ func getPermissions(ps []string) ([]string, []string) {
 	return permissions, features
 }
 
+func hasForegroundPermission(permissions []string) bool {
+	if fgPerms, ok := AndroidPermissions["foreground"]; ok {
```

Comment:
Isn't ok always true?

Reply:
I thought it should only be true when the package includes that import line?